config/locales/en.yml

```diff
@@ -1104,11 +1898,285 @@ en:
+      confirm_account_deletion:
+        body: <p> Your account in %{school_name} has been successfully deleted. </p> <p> All your personal information has been removed and you will no longer have access to this school.</p>
+      delete_account_token:
+        body: "<p>We received a request to delete your account in %{school_name}.</p> <p> Please click this %{link_to} to confirm account deletion. The link is valid for 30 minutes only.</p> <p> If you're unable to click the link above, please copy and paste the following link into your browser's address bar: </p>"
+        link_to: link
+        body_note: <p> This action is irreversible, and will permanently remove your access to the school.</p>
+      new_post:
+        body: <p>%{author_name} has posted a reply to something you said on the %{community_name} community.</p> <p>You can %{link_to}.
+        link_to: view the reply on the %{community_name} community
+      send_login_token:
+        body: <p> Please %{link_to} to login to your %{school_name} account.  </p>
+        link_to: click here
+      send_reset_password_token:
+        body: <p> We received a request to reset your %{school_name} password.  </p> <p> Please %{link_to} to change your password.  </p>
+        link_to: click here
+      daily_digest:
+        body: "There %{article_pending_submissions_count} <strong>%{pending_submissions_count}</strong> new submissions to review in %{courses_number} courses that you are a coach in:"
+        assigned_none: (none of which are assigned to you)
+        assigned: (%{stats} assigned to you)
+        latest_topics: "Latest topics posted on your communities:"
+        views: views
+        replies: replies
+        older: "Older, popular topics that have seen new activity:"
+        asked: asked %{days_ago} days ago.
+        control_emails: You can control these emails from %{link_to}.
+        link_to: your profile edit page
+  mailers:
+    applicant:
+      verify: Verify Your Email Address
+    coach:
+      added: You have been added as a coach in %{course_name}
+    course_author:
+      added: You have been added as an author in %{course_name}
+    course_export:
+      export: Export of %{course_export} course is ready for download
+    issued_certificate:
+      awarded: You have been awarded a certificate!
+    school_admin:
+      added: New School Admin Added
+      import: Import of Students Completed
+    startup:
+      new_feedback: New feedback from %{startup_feedback} on your submission
+    student:
+      added_as: You have been added as a student in %{school_name}
+    user:
+      new_reply: New reply for your post
+      daily_digest: Daily Digest
+      delete_account: Delete account from %{school_name}
+      deleted_success: Account deleted successfully from %{school_name}
+      deleted_in: Your account in %{school_name} will be deleted in 30 days
+    user_session:
+      log_in: Log in to
+      recovery: account recovery
```

Comment:
All of these appear to be subjects of the corresponding mailer method. Could we stick to naming them by the caller mailer method? I.e., instead of `mailer.applicant.verify`, use `mailer.applicant.enrollment_verification_subject` so that it's clearer what the translation is supposed to be _of_.

Reply:
Agreed @harigopal , I updated in this [commit ](https://github.com/pupilfirst/pupilfirst/pull/933/commits/9c6a71042082922b3ff0fa28b50031bb76cf26d4)